web/packages/teleport/src/services/api/api.test.ts

```diff
@@ -16,14 +16,117 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import api from './api';
+import api, { MFA_HEADER, defaultRequestOptions, getAuthHeaders } from './api';
+
+describe('api.fetch', () => {
+  const mockedFetch = jest.spyOn(global, 'fetch').mockResolvedValue({} as any); // we don't care about response
+
+  const mockedWebauthnResp = {
+    id: 'some-id',
+    type: 'some-type',
+    extensions: {
+      appid: false,
+    },
+    rawId: 'some-raw-id',
+    response: {
+      authenticatorData: 'authen-data',
+      clientDataJSON: 'client-data-json',
+      signature: 'signature',
+      userHandle: 'user-handle',
+    },
+  };
+
+  const mockecCustomOpts = {
```

Comment:
```suggestion
  const mockedCustomOpts = {
```

I don't think it's "mocked" though in any way, it's just some predefined test data that we pass directly to the function under tests, isn't it? "Mocked" indicates to me that it gets returned from some mocked function or something but that doesn't seem to be the case.

I think we could name both variables just `webauthnResp` and `customOpts`.